thonny/plugins/micropython/pip_gui.py

```diff
@@ -86,15 +86,15 @@ def _show_instructions_about_target(self):
 
     def _show_extra_instructions(self):
         self._append_info_text("\n\n")
-        self._append_info_text("Under the hood " + "\n", ("caption", "right"))
+        self._append_info_text(tr("Under the hood") + "\n", ("caption", "right"))
         self._append_info_text(
-            "This dialog uses `pipkin`, a new command line tool for managing "
-            "MicroPython and CircuitPython packages. ",
+            tr("This dialog uses `pipkin`, a new command line tool for managing "
+            "MicroPython and CircuitPython packages."),
             ("right",),
         )
         self._append_info_text("See ", ("right",))
```

Comment:
I suppose you have omitted "See .... for more info" because of the piecewise placement of the whole sentence. You could still make it translatable by translating and splitting like this 

```python
trans = tr("See %s for more info")
prefix = trans.split("%s")[0].strip()
suffix = trans.split("%s")[1].strip()
```

Reply:
My original idea was just about making "visible strings" translatable without any big effort ;) I can take a closer look (also about how tkinter works).